qemu/tests/cfg/fio_windows.cfg

```diff
@@ -1,16 +1,29 @@
 - fio_windows:
     type = fio_windows
     only Windows
+    image_size_disk1 = 2G
+    create_image_disk1 = yes
+    force_create_image_disk1 = yes
     cmd_timeout = 600
-    log_file = "C:\fio_log.txt"
     image_snapshot = yes
+    disk_index = 1
+    disk_letter = I
+    format_disk = yes
```

Comment:
9~10 lines should for data disk only

Reply:
Hi shuang,

Thanks for your advise, has changed the file as your comment, could you help to review it again? 

Thanks~